server/services/RunService.js

```diff
@@ -105,17 +105,24 @@ const db = require('../models/index');
  */
 
 /* eslint-disable no-unused-vars */
-async function configureRuns({
-    test_version_id,
-    apg_example_ids,
-    at_browser_pairs
-}) {
+async function configureRuns(
+    { test_version_id, apg_example_ids, at_browser_pairs },
+    commit = true
+) {
     try {
+        await db.sequelize.query('BEGIN;');
```

Comment:
This was the only way I could create a transaction that was compatible with the `ROLLBACK` in `dbCleaner`

Reply:
Unless we want to change the way we do dbCleaner, I think we might as well run with this -- but I agree it's unsatisfying!